appendix-basic65.tex

```diff
@@ -4375,20 +4455,19 @@ \subsection{FOR}
 % **********
 
 \newpage
-\label{foregroundcommand}
 \subsection{FOREGROUND}
 \index{BASIC 65 Commands!FOREGROUND}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$FE \$39
 \item [Format:] {\bf FOREGROUND} colour
-\item [Usage:] Sets the foreground colour
-               (text colour) of the screen to the argument,
-               which must be in the
-               range of 0 to 31.
-               Refer to the table under
-               {\bf BACKGROUND} on page \pageref{colourtable}
-               for the colour values and their corresponding colours.
-\item [Remarks:] {\bf COLOR} also has the ability to change the foreground colour.
+\item [Usage:] Sets the foreground text colour for subsequent {\bf PRINT} commands.
+
+               {\bf colour} the palette entry number, in the range 0 -- 255
```

Comment:
as before with COLOR, I think the real range is 0-31

Reply:
Fixed.